README.md

```diff
@@ -43,7 +43,8 @@ bazel build -c opt //:nighthawk
 ```
 USAGE:
 
-bazel-bin/nighthawk_client  [--jitter-uniform <duration>] [--open-loop]
+bazel-bin/nighthawk_client  [--label <<string>>] ... [--jitter-uniform
```

Comment:
Curious, why do we have some of the types with double and some with single angle brackets? Is this significant? Do we use ti to indicate that the option accepts multiple items?

Reply:
Hmm; double checking we pass this in via a TCLAP argument. It looks like that in some cases I wrapped the type description in `<>`, which is something TCLAP does by itself as well, leading up to double angle brackets. 
I backed it out here in 3cb9f42, and filed #256  for fixing this in other spots. Thanks for catching this.